src/app/shared/components/Field/index.tsx

```diff
@@ -0,0 +1,62 @@
+import Image from 'next/image';
```

Comment:
we can remove the `;` here and on line 3

Reply:
yep think it autocompletes them in when I hit tab (need to eslint up in this bitch!). Removed them